testdata/ecosystem/configmaps/s2i-nodejs.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: image.openshift.io/v1
```

Comment:
This yaml file is in `configmaps` directory but it actually contains an image stream and a deployment config. Please split it into two files and place it into correct directories.

Reply:
Done